spec/controllers/users_controller_spec.rb

```diff
@@ -16,21 +16,6 @@
       expect(response.code).to eq "200"
       expect(assigns[:user]).to be_new_record
     end
-
-    context "ENV has disabled sign-ups" do
-      before do
-        allow(ENV).to receive(:[]).and_return(nil)
```

Comment:
Here it is: https://github.com/codeforamerica/michigan-benefits/blob/2fd84508c7c64a50b1242ea9bb13de7d50de02ce/spec/support/generic_helper.rb#L4